project_stage_closed/README.rst

```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=========================
+Project Task Stage Closed
+=========================
+
+Odoo provides a 'closed' flag on project task stages, but this flag
+is in the sale_service module, which in turns pulls a lot of dependencies
+such as sale, accounting, and procurement. In many circumstances,
+it is desirable to have such a flag to indicate a task is closed,
+without needing so many dependencies.
+
+This module provides the flag in a way that is compatible with sale_service
+but depending only on the project module.
+
+Installation
+============
+
+There are no specific installation instructions for this module.
+
+Configuration
+=============
+
+There are no specific installation instructions for this module.
+
+Usage
+=====
+
+To use this module, you need to:
+
+ * go to Project > Configuration > Stages > Task Stages where
+   the Closed flag is available on the form view.
+
+ 
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/140/8.0
+
+Known issues / Roadmap
+======================
+
+When `Odoo PR 8186 <https://github.com/odoo/odoo/pull/8186>`_ 
```

Comment:
@rafaelbn I agree with @elicoidal. Since this impacts structure, it does not fit for OCB, as a module expecting the closed flag in the project module would work with OCB only.